src/components/floweditor/FlowEditor.tsx

```diff
@@ -277,13 +277,22 @@ export const FlowEditor = (props: FlowEditorProps) => {
         title={dialogMessage}
         alignButtons="center"
         skipCancel
-        buttonOk="Okay"
+        buttonOk="Take Over"
+        buttonMiddle="Go Back"
         handleOk={() => {
+          getOrganizationServices();
+          setCurrentEditDialogBox(false);
+        }}
+        handleMiddle={() => {
           setConfirmedNavigation(true);
           history.push('/flow');
         }}
       >
-        <p className={styles.DialogDescription}>Please try again later or contact the user.</p>
+        {/* <p className={styles.DialogDescription}>Please try again later or contact the user.</p> */}
```

Comment:
Remove comment if not needed.

Reply:
Removed this with this [Commit](https://github.com/glific/glific-frontend/commit/570664b36f545887015cf12542321f3a325be985)